recipes/install_binary.rb

```diff
@@ -8,9 +8,11 @@
 
 include_recipe 'ark'
 
+node.default['packer']['zipfile'] = "packer_#{node['packer']['version']}_" \
+  "#{node['os']}_#{node['packer']['arch']}.zip"
```

Comment:
I don't think this should be in here. This could (should?) be a default value for the new `zipfile` attribute in de `attributes.rb` file.